### PR TITLE
Fixing go cloud run CI

### PR DESCRIPTION
**Pull request by simonz130**

### CODE REVIEW

File: golang/.ci/go-cloud-run-hello-world.cloudbuild.yaml [hunk 1]
```diff
@@ -3,6 +3,7 @@ steps:
 - id: 'Build Container Image'
   name: 'gcr.io/cloud-builders/docker:latest'
   args: ['build', '-t', 'gcr.io/${PROJECT_ID}/${_SERVICE_TAG}:${COMMIT_SHA}', '.']
+  dir: 'golang/go-cloud-run-hello-world'
```
Comment (grayside) [inline golang/.ci/go-cloud-run-hello-world.cloudbuild.yaml:6]: The instructions in golang/CONTRIBUTING.md are for this to be run from the golang directory. Is it possible to run the trigger in that same context?

Comment (simonz130) [reply golang/.ci/go-cloud-run-hello-world.cloudbuild.yaml:6]: I tried to look at the trigger to see if you can specify the context outside of the yaml file and you can't. Hence this change.

**Comment by simonz130** [comment]
@grayside pretty ping

**Pull request merged**